Makefile

```diff
@@ -33,6 +34,7 @@ test-tier0:
 test-tier1:
 	$(MAKE) test-metrics
 	TIER1=1 $(MAKE) test-analysis
+	$(MAKE) test-jira
```

Comment:
@aufi it cannot be under `test-tier1` because the secrets are accessible only from non-forked repo, which means that `main-tier1` will fail for each created PR since the workflow is triggered from a forked repo

This is why it was on `nightly-tier1.yaml`

Reply:
Thanks for clarification, will update.